bundler.d/test.rb

```diff
@@ -5,8 +5,11 @@
   gem 'minitest-reporters', '~> 1.4', :require => false
   gem 'minitest-retry', '~> 0.0', :require => false
   gem 'minitest-spec-rails', '~> 6.0'
+  # Lock rack-test until some of below deps are fixed
+  gem 'rack-test', '< 2'
```

Comment:
Note this is actually a runtime dependency (actionpack pulls it in). I wonder if the pin should live at the top level.

Reply:
Looks like it's `show_me_the_cookies`: https://github.com/nruth/show_me_the_cookies/pull/64.